Toolbar: Add a Getting Started tutorial (#3795)
DevExpress/devextreme-documentation: concepts/05 UI Components/Toolbar/00 Getting Started with Toolbar/00 Getting Started with Toolbar.md

#include tutorials-intro-installationnote

> The Toolbar contains items that manage the page content.

Reviewer: The phrase "that manage the page content" limits the application of this component. I assume that items can do anything... Maybe they can add a music album to the library. This action doesn't necessarily change the current page's content. 

I don't really have a good replacement or edit for this sentence. I would simply remove it.

> This tutorial shows how to add the Toolbar component to your application and configure its core features.

Reviewer: If you instruct customers to add something new to the app, or create something new, we usually use indefinite article. 

```suggestion
This tutorial shows how to add a Toolbar component to your application and configure its core features.
```

<div class="simulator-desktop-container" data-view="/Content/Applications/21_2/GettingStartedWith/Toolbar/index.html, /Content/Applications/21_2/GettingStartedWith/Toolbar/index.js, /Content/Applications/21_2/GettingStartedWith/Toolbar/index.css"></div>

Each section in this tutorial describes a single configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-devextreme-toolbar" target="_blank">getting-started-with-toolbar</a>.

> The Toolbar consists of four sections: left area, central area, right area, and overflow menu. Follow the list below to learn how to add items to the Toolbar:

Reviewer: You create a table of contents (at least it looks like a table of contents). But it's missing one item then - Customize Items. Maybe it's better to include it to avoid confusion... Or maybe you rely on some side navigation? 

Just to be consistent, I guess you can change the introduction like this: 

```suggestion
You can add items to the toolbar in its four sections: left area, central area, right area, and overflow menu. Templates help you customize items displayed in any of these section. Refer to the following tutorial steps for details:
```

And then you can add another item about templates (Customize Items).

- The central area   
[Create a Toolbar and its Items](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/#Create_a_Toolbar_and_its_Items)

> - The left and right area   

Reviewer 2: ```suggestion
- The left and right areas   
```

#include tutorials-intro-installationnote

This tutorial shows how to add a Toolbar component to your application and configure its core features.

<div class="simulator-desktop-container" data-view="/Content/Applications/21_2/GettingStartedWith/Toolbar/index.html, /Content/Applications/21_2/GettingStartedWith/Toolbar/index.js, /Content/Applications/21_2/GettingStartedWith/Toolbar/index.css"></div>

Each section in this tutorial describes a single configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-devextreme-toolbar" target="_blank">getting-started-with-toolbar</a>.

> You can add items to the Toolbar in its four sections: left area, central area, right area, and overflow menu. Templates help you customize items displayed in any of these section. Refer to the following tutorial steps for details:

Reviewer 3: ```suggestion
You can add items to the Toolbar in four areas: left area, central area, right area, and overflow menu. Templates help you customize items displayed in any of these sections. Refer to the following tutorial steps for details:
```

- The central area   
[Create a Toolbar and its Items](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/#Create_a_Toolbar_and_its_Items)

- The left and right areas   
[Specify Item Location](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/#Specify_Item_Location)

- The overflow menu   
[Create an Overflow Menu](/Documentation/Guide/UI_Components/Toolbar/Getting_Started_with_Toolbar/#Create_an_Overflow_Menu)

> - The item customization   

Reviewer: "Item customization" is a general process, not a specific "action". You could maybe say "the item customization you carried out in the previous step". You refer to a specific customization. But when you talk about customization as a process, in general, you don't need an article. 

By the way, I'd probably remove articles in all these list items. These titles - like "Overflow menu" - don't have to include articles.

 ```suggestion
- Item customization   
```
